packages/react-cli/src/commands/generateMaterial.ts

```diff
@@ -0,0 +1,47 @@
+import Command from '@oclif/command';
+import { ensureMaterialExists } from '../tasks/ensure-material-exist';
+import { runEmberInteractively } from '../tasks/run-ember';
+import inquirer = require('inquirer');
+
+export class GenerateMaterialCommand extends Command {
+    static description = 'Creates components with Material Components installed';
+
+    static aliases = ['material', 'm'];
```

Comment:
if material is an alias,
what prints when you do:
```bash
react --help
```

Reply:
![image](https://user-images.githubusercontent.com/7727018/56431061-966f3e00-6296-11e9-8826-b8c04f2709cf.png)

Reply:
from a user perspective, do you want to type generateMaterial? or do you want to type material?

from a library maintainer perspective, what do you think is the most natural?

Reply:
I was going back and forth on this... I would say I would want to type material. I was thinking since we are utilizing the ember generate command need the name be more descriptive.

Reply:
Sounds good!